r100678 | fanf | 2002-07-25 10:59:40 -0500 (Thu, 25 Jul 2002) | 5 lines

Document the FreeBSD default for CheckHostIP, which was changed in
rev 1.2 of readconf.c.

--- ssh_config.5.orig	2010-08-04 21:03:13.000000000 -0600
+++ ssh_config.5	2010-09-14 16:14:13.000000000 -0600
@@ -164,7 +164,7 @@
 .Dq no ,
 the check will not be executed.
 The default is
-.Dq yes .
+.Dq no .
 .It Cm Cipher
 Specifies the cipher to use for encrypting the session
 in protocol version 1.
